make/uncrustify.cfg

```diff
@@ -0,0 +1,147 @@
+# run with FLIGHT=some_file.c make uncrustify_flight
+
```

Comment:
@mlyle do you know of a handy online calculator for visualizing the effects of this uncrustify configuration?